index.html

```diff
@@ -0,0 +1,227 @@
+  </head>
+  <body class="page">
+    <header class="header" id="home">
+      <nav class="navigation header__navigation">
+        <div class="logo navigation__logo">
+          <a href="/" class="logo__link navigation__item">
+          <img class="logo__image" src="./src/images/logo-bose.svg" alt="BOSE logo">
+          </a>
+        </div>
+        <div class="icons navigation__item navigation__icons">
+          <a href="tel:+1 234 555-55-55" class="phone">
+          <span class="phone__label">+1 234 555-55-55</span> 
+          <img class="phone__image" src="./src/images/icon-phone.svg" alt="Phone">
+          </a>
+
+          <a href="#menu" class="burger navigation__item icons__burger">
+            <span class="burger__label">menu</span>
+          </a>
+        </div>
+      </nav>
+      <h1 class="title header__title title--header">
+        The world shades.<br>
+        Your music shines.
+      </h1>
+    </header>
+
+    <div class="menu" id="menu">
+      <nav class="navigation menu__navigation">
+        <a href="#" class="logo navigation__item">
+          <img class="logo__image" src="./src/images/logo-bose.svg" alt="BOSE logo">
+        </a>
+
+        <a href="#" class="navigation__item--hidden"
+          <img class="phone__image" src="./src/images/icon-phone.svg"
+            alt="Phone">
+        </a>
+
+        <a href="#" class="close navigation__item">
+          <img class="close__image" src="./src/images/icon-menu-close.svg" alt="Close navigation">
+        </a>
+      </nav>
+
+      <nav class="links menu__links">
+        <ul class="links__list">
+          <li class="links__item"><a class="links__link" href="#home">Home</a></li>
+          <li class="links__item"><a class="links__link" href="#recommended">Recommended</a></li>
+          <li class="links__item"><a class="links__link" href="#categories">Categories</a></li>
+          <li class="links__item"><a class="links__link" href="#howtobuy">How to buy</a></li>
+          <li class="links__item"><a class="links__link" href="#contacts">Contacts</a></li>
+        </ul>
+
+        <p class="menu__paragraph">+1 234 555-55-55</p>
+        <p class="menu__paragraph">
+          <a class="menu__call-link" href="tel:+1 234 555-55-55">call to order</a>
+        </p>
+      </nav>
+    </div>
+
+    <main class="main" id="recommended">
+      <section class="section main__section">
+        <h2 class="title title--secondary main__title">
+          Recommended
+        </h2>
+
+        <div class="recommended section__recommended">
+          <article class="product recommended__article">
+            <img src="./src/images/photos/product-01.png" class="product__image" alt="Bose portable Smart speaker">
+            <p class="product__name">Bose portable Smart speaker</p>
+            <p class="product__category">Smart home</p>
+            <p class="product__price">$ 399.00</p>
+          </article>
```

Comment:
This for example shoule looke like this:
```suggestion
          <article class="product recommended__article">
            <img 
                src="./src/images/photos/product-01.png" 
                class="product__image" 
                alt="Bose portable Smart speaker"
            >
            
            <p class="product__name">Bose portable Smart speaker</p>
            
            <p class="product__category">Smart home</p>
            
            <p class="product__price">$ 399.00</p>
          </article>
```